pages/Reference/glossary.md

```diff
@@ -6,20 +6,17 @@ menu: Reference
 
 # Glossary
 
-* **Account** - See "Social Identity"
-* **Announcement** - An item of data, typically an image or note, posted to the blockchain
-* **Publisher** - The contract that posts new content from a publisher to the blockchain through batch publications
+* **Activity Content** - The linked content connected to an "Announcement"
+* **Announcement** - An item of data, typically an image or note, posted to the blockchain via a "Batch Publication"
 * **Bad actor** - A user intentionally using the application for bad-faith or illegal purposes
+* **Batch Publication** - A collection of Announcements
 * **Bot** - An automated account, sometimes malicious but often providing some service ("Hey Alexa, what's the weather?")
 * **Consumer** - Someone who reads content from social media
 * **Contract address** - The unique number associated with a smart contract posted on the blockchain
-* **Hash** - A string of characters generated from a hash function, see "Hash function"
-* **Hash function** - A cryptographic function whose output is effectively unique for any given input without any information from the input being accessible in the output
 * **Hash Collision** - The vanishingly unlikely possibility of a hash providing the same output for two different inputs
-* **Producer** - Someone who creates content on a social media
-* **Prosumer** - Someone who is both a producer and a consumer, see "Consumer" and "Producer"
-* **Social Identity** - A representation of a user, bot or group online, see "User," and "Bot"
-* **Social Identity Address** - The contract address of the smart contract we use to represent a social identity in our system, see "Contract address" and "Social Identity"
+* **Hash function** - A cryptographic function whose output is effectively unique for any given input without any information from the input being accessible in the output
+* **Hash** - A string of characters generated from a hash function, see "Hash function"
+* **Identity Contract Address** - The contract address of the smart contract we use to represent a user's delegations identity in our system, see "Contract address"
```

Comment:
A little unclear to me what we are trying to say here - are we trying to say this contract represents a `user's delegated identities`?

Reply:
```suggestion
* **Identity Contract** - The smart contract used to store a user's delegated public addresses, see "Contract address"
```